src/common/slack.ts

```diff
@@ -261,6 +261,37 @@ export const notifyCommentReport = async (
   });
 };
 
+export const notifyReportUser = async (
```

Comment:
Looks to be almost exactly the same as others. I wonder if we can create a generic function for this. Not related to this PR, just noticed.